packages/api-client/src/connections/CentralApi.ts

```diff
@@ -63,6 +63,13 @@ export class CentralApi extends BaseApi {
     }
   }
 
+  public createSurveyResponse(
+    response: MeditrakSurveyResponseRequest,
+    queryParameters?: QueryParameters,
+  ): Promise<SurveyResponseCreatedResponse> {
+    return this.connection.post('surveyResponse', queryParameters, response);
```

Comment:
I thought it was appropriate to add the createSurveyResponse even though there is already a createSurveyResponses since it is such core functionality. Note that I also return the response from central-server so that we know what the survey response id is.